build.gradle

```diff
@@ -497,6 +497,7 @@ configurations {
             force "org.apache.httpcomponents:httpcore:4.4.16"
             force "com.google.errorprone:error_prone_annotations:2.23.0"
             force "org.checkerframework:checker-qual:3.40.0"
+            force "ch.qos.logback:logback-classic:1.3.12"
```

Comment:
@cwperks this should probably be set to `1.2.13` since Logback 1.3.x needs SLF4J 2.0 APIs

Reply:
Thank you reta! How can we catch issues like this as part of CI?

From logback's [README](https://github.com/qos-ch/logback#java-ee-and-jakarta-ee-versions):

> Both 1.3.x and 1.4.x series require SLF4J 2.0.x or later.

Reply:
Is the issue preventing slf4j upgrade that OpenSearch still supports Java 8? From the slf4j [README](https://github.com/qos-ch/slf4j#how-to-build-slf4j)

> SLF4J version 2.0.x will run under Java 8 but requires Java 9 or later to build.

Reply:
> Is the issue preventing slf4j upgrade that OpenSearch still supports Java 8?

We basically could move to slf4j 2.x but I believe some deps (like Jetty 9 fe, if I am not mistaken)  do still rely on slf4j 1.x